.github/workflows/cache-speed-test.yml

```diff
@@ -47,7 +48,7 @@ jobs:
           path: |
             modules/
             zephyr/
```

Comment:
These will need to be updated as well for them to be cached properly.
```
~/zephyrproject/zephyr
~/zephyrproject/modules
```
should work.